src/main/java/dao/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
```

Comment:
The class starts with an empty line, which is against the checklist rules. Please remove this redundant empty line.